### DISCUSSION TITLE
Suggestion: Add a global variable for converting MeV to Joules to be used by all applications

### DISCUSSION THREAD

**Discussion by MartinMartinov** [nrc-cnrc/EGSnrc#701] [Ideas]
I'm writing my own usercode right now, and I think that if the egs++ code had a global definition for MeV to J conversion (similar to the global epsilon threshold), it would be easier to ensure consistent scoring results in Gy between codes.  It might also be a good idea to split it into two factors:

```
EGS_Float MeVtoJ = 1.6021773e-10; // Pure energy conversion
EGS_Float MeVtoGy = MeVtoJ/1000; // Convert energy, and mass more gram to kilogram
```

As the latter conversion is the factor used in most of the codes I've used and would be easier to plug into egs_chamber and egs_brachy, for example.

**Comment by mainegra** [comment]
I love this suggestion Martin!

**Comment by ftessier** [comment]
I will integrate this along with #676.
